modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -56,52 +56,61 @@ module.exports = {
       bold: {
         component: 'AposTiptapButton',
         label: 'Bold',
-        icon: 'format-bold-icon'
+        icon: 'format-bold-icon',
+        command: 'toggleBold'
       },
       italic: {
         component: 'AposTiptapButton',
         label: 'Italic',
-        icon: 'format-italic-icon'
+        icon: 'format-italic-icon',
+        command: 'toggleItalic'
       },
       underline: {
         component: 'AposTiptapButton',
         label: 'Underline',
-        icon: 'format-underline-icon'
+        icon: 'format-underline-icon',
+        command: 'toggleUnderline'
       },
-      horizontal_rule: {
+      horizontalRule: {
         component: 'AposTiptapButton',
         label: 'Horizontal Rule',
-        icon: 'minus-icon'
+        icon: 'minus-icon',
+        command: 'setHorizontalRule'
       },
       link: {
         component: 'AposTiptapLink',
         label: 'Link',
         icon: 'link-icon'
       },
-      bullet_list: {
+      bulletList: {
```

Comment:
we'll need to update our demo and boilerplate.

Reply:
drafted
https://github.com/apostrophecms/a3-boilerplate/pull/22
https://github.com/apostrophecms/a3-demo/pull/81